citation.php

```diff
@@ -39,7 +39,7 @@ function get_citation_by_id($id)
 {
     $pos = strpos($id, '/');
     $dcoid = substr( $id, $pos + 1 ) ;
-    $searchUrl = "http://localhost:49200/dco/publication/_search?q=_id:$dcoid";
```

Comment:
This needs to be switched back before we merge the pull request. Probably would work, but since it's running on the local machine we need to eliminate the http proxy.